AutomatedTesting/Gem/PythonTests/Prefab/tests/spawnables/Lua_Spawnables_EntityClearedOnGameModeExit.py

```diff
@@ -43,12 +43,14 @@ def validate_entities_in_edit_mode(condition):
     assert spawner_entity, "Failed to find Spawner entity at runtime"
     assert flower_entity, "Failed to find PinkFlower entity at runtime"
 
+'''
```

Comment:
Need a reason for this being commented out here, a link to the bug tracking the fix for this would be useful. Also, as a nitpick, we shouldn't block comment out with quotes, since it's technically how Python finds docstrings: https://peps.python.org/pep-0008/#block-comments